internal/commands/sbomtest/sbomtest.go

```diff
@@ -85,13 +86,24 @@ func TestWorkflow(
 		return nil, err
 	}
 
-	data, contentType, err := NewPresenter(ictx).Render(TestResult{
-		Summary: TestSummary{
-			TotalVulnerabilities: results.Summary.TotalVulnerabilities,
-		},
-	})
+	var ct string
+	var buf bytes.Buffer
+
+	if ictx.GetConfiguration().GetBool("json") {
+		if err := RenderJSONResult(&buf, results); err != nil { //nolint:govet // Just shut up.
+			// TODO: replace with calls to error factory
```

Comment:
Can the errors not be added as part of this PR?

Reply:
done